community/import-tool/src/test/java/org/neo4j/tooling/QuickImport.java

```diff
@@ -98,12 +94,17 @@ public int denseNodeThreshold()
                 return args.getNumber( dense_node_threshold.name(), super.denseNodeThreshold() ).intValue();
             }
         };
-        Input input = new CsvDataGeneratorInput(
-                nodeHeader, relationshipHeader,
-                COMMAS, nodeCount, relationshipCount, new Groups(), idType, labelCount, relationshipTypeCount,
-                silentBadCollector( 0 ));
+
+        SimpleDataGenerator generator = new SimpleDataGenerator( nodeHeader, relationshipHeader, relationshipCount,
+                nodeCount, labelCount, relationshipTypeCount, idType );
```

Comment:
The `relationshipCount` argument is slotted into the `randomSeed` parameter, which doesn't sound like it does the same thing.